### ISSUE TITLE
How to avoid output blank files on minifying errors?

### ISSUE THREAD

**Issue by EthraZa** [tdewolff/minify#560]
For some files that are unable to be minifeid, like 

**Test.js:**
`alert('hi);`

minify will show:
```
ERROR: cannot minify Test.js: unterminated string literal in arguments on line 2 and column 7
    2: alert('hi);
```
and will generate a blank file at output folder.

Is there any option to "**copy file as is on minify error**"?

**Comment by EthraZa** [comment]
OR, can the **--match** switch be used to create an exclusion list?
Like in: **sync** all files, minifying ***.js** but not **\*Test\*.js**

**Comment by tdewolff** [comment]
In case of errors it should just output the original file, I agree. As I'm
on vacation, I can make this change in early March!

**Comment by tdewolff** [comment]
Fixed now. Perhaps `--include` and `--exclude` filters could be added much like `rsync`, but on the other hand I feel like the tool should be simple and "do one thing well" kind of Linux philosophy. The latter would push to remove `--match` and let users use the `find` tool for that, but I think it's better to be pragmatic in this case :-)

**Comment by EthraZa** [comment]
I have even printed and glued to my monitor a KISS principle reminder so I don't get too creative and do stupid things. :)
So, I understand and agree to the keep it simple. 

But, look at my case:
Right now, what I'm trying to do is, sync my Dev folder to my Prod folder, minifying the JS files, without removing Prod folder beforehand so I can keep files in Prod that are not in Dev (yes I know, and don't want to touch that mess), and avoid the find tool to do that because it is slow as hell and take a hour or so to do what minify do in a couple minutes.
Since it is messing some JS files, I even have one already minified that is get minified again (not blanking out) but is getting corrupted some how, so, as we don't have an exclude list, I had to call minify from a shell script that cp this messed files to Prod after minify is done its job. 
What I'm trying to say is that what could be a simple call to minify, become a call to a script that uses minify and need to be checked after run to be sure no new files at php vendor folders get messed up. Sometimes what get simple at devs get hard at users. hehe

But just saying, not complaining. Minify is a great tool anyways and you are always very responsive at fixing eventual bugs. 
But, I got say, that creative day you added the sync param to minify, you take it from cp level to rsync level and now you got new adventures. :D
